aeron-cluster/src/main/java/io/aeron/cluster/ClusterTool.java

```diff
@@ -613,6 +620,66 @@ public static void describe(final PrintStream out, final ClusterMarkFile[] servi
         }
     }
 
+    /**
+     * Determine if the given node is a leader
+     *
+     * @param out        to print the output to.
+     * @param clusterDir where the cluster is running.
+     * @return           0 if the node is an active leader in a closed election, 1 if not,
+     *                   -1 if the mark file does not exist.
+     */
+    public static int isLeader(final PrintStream out, final File clusterDir)
+    {
+        if (markFileExists(clusterDir))
+        {
+            try (ClusterMarkFile markFile = openMarkFile(clusterDir, System.out::println))
+            {
+                final String aeronDirectoryName = markFile.decoder().aeronDirectory();
+                final AtomicLong nodeRoleCounter = new AtomicLong(-1);
```

Comment:
`MutableLong` is sufficient for these variables.

Reply:
Thanks; I've pushed a change, switching to `MutableLong`.